src/tree_schema.h

```diff
@@ -2146,7 +2171,7 @@ const char *lys_nodetype2str(uint16_t nodetype);
  */
 const char *lyxp_get_expr(const struct lyxp_expr *path);
 
-/** @} schematree */
+/* @} schematree */
```

Comment:
Again a typo? All doxygen comments should start with `/**` AFAIK.

Reply:
again, forgotten debugging change, fixed